docs/developer/architecture/ci.rst

```diff
@@ -0,0 +1,120 @@
+CI
+==
+
+Pre merge
+---------
+
+What should be tested in pre-merge on every ``user/*``, ``feature/*``,
+``improvement/*``, ``bugfix/*``, ``w/*`` branches.
+
+- Building tests:
+
+  - Build
+  - Lint
+  - Unit tests
+
+- Installation tests
+
+  - Simple install RHEL
+  - Simple install CentOs + expansion
+
+When merging several PRs at the same time, so when we are on a ``q/*``
+branches, we may want to have additionnal test like:
```

Comment:
Could be rewritten `When merging several pull requests at the same time, given that we are on a queue branch (`q/*`), we may want to have or we may require additional tests like:`

Reply:
And maybe add a reason why we require these additional tests